model/accounts_mgmt/v1/subscriptions_resource.model

```diff
@@ -55,6 +55,10 @@ resource Subscriptions {
 		//
 		// If the parameter isn't provided, or if the value is empty, then the order of the
 		// results is undefined.
+		@http(name = "orderBy")
```

Comment:
note that this was required to build the http param correctly, without it the param was being generated in pascal case: `OrderBy`